.config/lingua.dic

```diff
@@ -0,0 +1,88 @@
+90
```

Comment:
Just curious, where'd you get this list from?

Also, what's with the `/<letter>` at the end of certain words?

Reply:
It is kinda...it is called `Hunspell` and by that letters it is allowed to specify kind of dictionary like "en_US". Paste from the documentation:

> If the .dic files contains incorrect rules for a word, then add the word with the correct rules to .dic_delta. For example, if en_US.dic contains the entry raccoon/M, but the rule should be /MS, then add raccoon/MS to en_US.dic_delta. Rules for entries in .dic_delta override the rules for entries in .dic.

Source: https://www.chromium.org/developers/how-tos/editing-the-spell-checking-dictionaries

What is the source of the list? Well, long story short, around 2 months ago I was thinking that the task will take me ~1h and started to do it, but it was not so simple and was needed to leave it, but I to follow and look at Parity's repositories and I found that PR: https://github.com/paritytech/polkadot/pull/1841/files and my dream is to do at least one merge/ed PR and when I saw the PR I immediately made my PR.

That is it, that's the story.

Reply:
> It is kinda...it is called `Hunspell` and by that letters it is allowed to specify kind of dictionary like "en_US". Paste from the documentation:
> 
> > If the .dic files contains incorrect rules for a word, then add the word with the correct rules to .dic_delta. For example, if en_US.dic contains the entry raccoon/M, but the rule should be /MS, then add raccoon/MS to en_US.dic_delta. Rules for entries in .dic_delta override the rules for entries in .dic.
> 
> Source: https://www.chromium.org/developers/how-tos/editing-the-spell-checking-dictionaries

I still don't fully get it, but good to know there's a source for the strangeness 😅

> What is the source of the list? Well, long story short, around 2 months ago I was thinking that the task will take me ~1h and started to do it, but it was not so simple and was needed to leave it, but I to follow and look at Parity's repositories and I found that PR: https://github.com/paritytech/polkadot/pull/1841/files and my dream is to do at least one merge/ed PR and when I saw the PR I immediately made my PR.
> 
> That is it, that's the story.

Haha, sounds good! Keep in mind that you will need to add some "bridge" specific words in there though (e.g our test networks Rialto and Millau). I'm not sure what other false positives you'll get, but I imagine there'll be a few.

Reply:
https://user-images.githubusercontent.com/1605705/113916972-eaa3bd80-97e0-11eb-9737-8097040cb69c.mp4

@HCastano  I wanted to check output from the commands like in recording but I got empty output, I have missed something all it works like supposed to work?